src/Components/PropertyInspector/PropertyInspector.tsx

```diff
@@ -42,6 +43,12 @@ type RelationshipPropertyInspectorProps = {
     resolvedTwin?: IADTTwin;
 };
 
+export type DataHistoryControl =
```

Comment:
using boolean value is softly enables the data history meaning the decision to show the control button is passed to the adapter itself (if it has the ADX connection information show the control button, if has not then hide the control button). But using the object with isEnabled flag, you can force the control component to show the control button no matter what, because we fetch the ADX connection information anyhow when the modal is open internally if the adapter does not have it yet.